tests/core/skin/skin.js

```diff
@@ -55,6 +55,12 @@ bender.test( {
 		var skin = CKEDITOR.skin;
 
 		var editor = new CKEDITOR.editor();
+
+		editor.element = {
```

Comment:
Seems like you already fixed the issue with missing editor element. If so, do we still require this dirty stub among unit tests?